server/tests-py/queries/v1/bulk/mixed_access_mode.yaml

```diff
@@ -3,15 +3,11 @@ url: /v1/query
 status: 400
 response:
   path: $
-  error: "incompatible access mode requirements in bulk query: $.args[1] requires READ WRITE, $.args[2] requires READ ONLY"
+  error: "incompatible access mode requirements in bulk query, expected access mode: Just READ WRITE but $.args[1] forces Just READ ONLY"
```

Comment:
Including `Just` in the error message doesn’t seem right to me—was that intentional?